src/Controller/PreciousUsersController.php

```diff
@@ -22,13 +22,35 @@ public function initialize()
         $this->PreciousUsersService = new PreciousUsersService($this->PreciousUsers);
     }
 
+    public function isAuthorized($user)
+    {
+        $action = $this->request->getParam('action');
+        // 一覧、追加はログインしているユーザーに許可する
+        if (in_array($action, ['index', 'new', 'add'] )) {
+            return true;
+        }
+
+        // 更新・削除対象が現在のユーザーに属していることを確認する
+        if (in_array($action, ['update'] )) {
```

Comment:
```suggestion
        if (in_array($action, ['update'], true)) {
```
in_arrayはできるだけtrue入れてあげたいです！

Reply:
trueを入れると型もチェックされるんですね！